### PR TITLE
Make redesign of list page

### PR DESCRIPTION
**Pull request by yuriidanyliak**
- [x] font-family: "Open Sans", serif  
- [x] colour of List name: #878787 (light grey)
- [x] colour of links: #0088cc (blue) (not bold!)

- [x] put list content into panel area with back-ground: #f6f7f7, inside the content colour is white
- [x] the list container should became similar to [sample item](http://img.in6k.com/screens/screense808f4cb0cc8.png), [Asana](https://blog.asana.com/) (we can use static image for cover before we add support of setting the images)

- [x] make font-size smaller: for items, links 

- [x] display date of the list creation like [this](http://img.in6k.com/screens/screens2919b8cb9e6e.png) using [Bootstrap](http://getbootstrap.com/components/#glyphicons-glyphs)

### CODE REVIEW

**Comment by sergeyradkevich** [comment]
@yuriidanyliak, Looks nice so far :+1:

**Comment by yuriidanyliak** [comment]
@sergeyradkevich Redeployed and added date displaying.

**Comment by sergeyradkevich** [comment]
Looks good! Thanks

**Comment by sergeyradkevich** [comment]
Can you please remove bold

**Comment by sergeyradkevich** [comment]
@yuriidanyliak, Let's decrease the height of the list image (cover) twice.

**Comment by yuriidanyliak** [comment]
@sergeyradkevich, the top half of the picture should be displayed, correct? Can I calculate the actual size of the image with JS or I should hardcode this CSS property near the half of the current?

**Comment by yuriidanyliak** [comment]
@sergeyradkevich, updated.

**Comment by yuriidanyliak** [comment]
@sergeyradkevich, is there some sense to refactor existing styles?

**Comment by yuriidanyliak** [comment]
@sergeyradkevich, updated.

**Comment by sergeyradkevich** [comment]
Alignment issue

File: config/settings/development.yml [hunk 1]
```diff
@@ -1,2 +1,6 @@
 notices:
-  title: Rolling RoR
\ No newline at end of file
+  title: Rolling RoR
+
+formats:
+  date:
+    long: '%B %-d, %Y'
```
Comment (sergeyradkevich) [inline config/settings/development.yml]: @yuriidanyliak, the format should not be duplicated, please find out the way to put the formats into common "group".

Comment (sergeyradkevich) [inline config/settings/development.yml]: I believe one of the options is to put all settings into one file with high-level keys for environments + have a common "group" of settings that is included into proper environments.

**Comment by sergeyradkevich** [comment]
@yuriidanyliak, Reviewed.

File: config/settings/development.yml (hunk 1 above)
Comment (yuriidanyliak) [inline config/settings/development.yml]: I also expected such ability but placing application.yml in `/config` and '/config/settings' (as was told in the README of handy gem) hadn't any effect: its content wasn't read to the Settings object in any of the cases/variants.

**Comment by yuriidanyliak** [comment]
@sergeyradkevich, updated & redeployed.

**Pull request merged**